pkgs/by-name/wh/whitesur-icon-theme/package.nix

```
@@ -32,13 +32,13 @@ lib.checkListOfEnum "${pname}: theme variants"
   stdenvNoCC.mkDerivation
   rec {
     inherit pname;
-    version = "2024-09-07";
+    version = "v2025-02-10";
```

```suggestion
    version = "2025-02-10";
```

https://github.com/NixOS/nixpkgs/blob/38e15cc875f4f7292d1732f77d7f882073a34865/pkgs/README.md?plain=1#L383-L385

---

Thanks for the suggested changes